### ISSUE TITLE
SRTM already includes height of buildings since it is a Digital Surface Model (DSM)

### ISSUE THREAD

**Issue by valantano** [domlysz/BlenderGIS#730]
I have a simple conceptual issue:
So the SRTM dataset is a Digital Surface Model (DSM) meaning that it includes the height of buildings.
Digital Terrain Models (DTMs) on the ohter hand only contain the bare-earth height.
So, if you would want to model a city in 3D, according to all tutorials and the GIF inside the README.md, you would create a basemap, then import the SRTM data from Open Topography and than add the buildings on top.
In my understanding this would probably work for regions with few buildings because of the bad resolution of the SRTM (30m or 1 arcsecond) but in cities the SRTM would already include the height of the buildings and the 3D models of the buildings would be placed on top of that.
Wouldnt it be better to use DTMs in order to work around this issue?
I can understand that the SRTM is probably still a good approximation but then it should be mentioned somewhere at least.
Then on the other side Skyscraper dont seem to be a big problem in th SRTM dataset.

**Comment by atseewal** [comment]
Whichever data set is used for elevation data, it will always have some level of uncertainty in the measurements. These elevation data sets usually have some level of accuracy associated with them (see figure 1 [here](https://www.frontiersin.org/articles/10.3389/feart.2018.00233/full)). Even the most accurate data still isn't perfect.

Blender is a visualization tool, so as long as it can easily import elevation that is pretty close to reality, I'd say it's working well. That being said, more options to easily use different elevation data wouldn't hurt, especially if SRTM isn't working to create the elevation map you're looking for.

_Side note_: I believe all of the SRTM data sets are based off data collected from a space shuttle mission back in 2000 (which was later reprocessed and released for different versions). So it's entirely possible some skyscrapers weren't there to throw the elevation off!

**Comment by valantano** [comment]
You are right. And your link did really help me in my research. Thanks for that.
Maybe a possibility would be to use Bare Earth DEM. This would improve SRTM at least for vegetated areas.
However, it is probably also possible to use OpenStreetMap and SRTM to correct the elevation in city areas.
